### PR TITLE
Output interactive mode env to a file

### LINKED ISSUE #518: Output interactive mode environment to a file

### PR DESCRIPTION
**Pull request by zakaprov**
Resolves #518 

This change makes interactive mode output its environment variables to a file (which can be `source`d), rather than print them out to the console. Example of the file's contents:
```
export PATH=/tmp/tmpx7o16vhf:$PATH
export YAGNA_APPKEY=a70facb9501d4528a77f25574ab0f11d
export YAGNA_API_URL=http://172.19.0.6:6000
export GSB_URL=tcp://172.19.0.6:6010
```

**Known limitations**
- there's no support for Windows, adding it would significantly increase complexity due to how `PATH` extensions are handled
- each time the file is `source`d the `gftp` script directory is appended to the `PATH` variable
- ~subnet tag currently cannot be set through an env variable (perhaps it should be possible?)~ **added to `yapapi` in https://github.com/golemfactory/yapapi/pull/546**

**Possible additions**
- ~revisit/improve extending the `PATH` variable~ **done**
- ~add an integration test for the interactive mode~ **done**
- ~mention the "new way" in readme/docs~ **done**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,14 +85,18 @@ python -m goth start your/output/dir/goth-config.yml
...
 If everything went well you should see the following output:
 ```
-Now run your requestor agent as follows:
+Local goth network ready!
 
-$ PATH=/tmp/...:$PATH YAGNA_APPKEY=3438...7901 YAGNA_API_URL=http://172.19.0.6:6000 GSB_URL=tcp://172.19.0.6:6010 examples/blender/blender.py --subnet goth
+You can now load the requestor configuration variables to your shell:
 
-Press Ctrl+C at any moment to stop the test harness.
+source /tmp/goth_interactive.env
```
Comment (azawlocki) [inline README.md:92]: I didn't raise this objection when we decided to put test logs in `/tmp/goth-tests` so maybe now it's too late to complain about it, but I don't like the idea of putting any files that can be accessed by user -- such as the log files, which are used as test artifacts, or this .env file -- in `/tmp`.

While I'm not proposing to change the location of the log files now, how about writing the .env file in the current working directory?

File: README.md [hunk 2]
```diff
@@ -85,14 +85,18 @@ python -m goth start your/output/dir/goth-config.yml
 
 If everything went well you should see the following output:
 ```
-Now run your requestor agent as follows:
+Local goth network ready!
 
-$ PATH=/tmp/...:$PATH YAGNA_APPKEY=3438...7901 YAGNA_API_URL=http://172.19.0.6:6000 GSB_URL=tcp://172.19.0.6:6010 examples/blender/blender.py --subnet goth
+You can now load the requestor configuration variables to your shell:
```
Comment (azawlocki) [inline README.md:90]: Why not leave the old way of starting the requestor scripts as an alternative? I'm quite used to it, I always just copy the line with the environment vars and replace the script path (and now I wouldn't have to add `--subnet goth`, cool!). In contrast to `source`'ing the `.env` file, the "old way" does not modify my current environment.

**Review by azawlocki — CHANGES_REQUESTED** [review]
I would be really grateful if you left the old way of starting the requestor script as an alternative to doing source'ing the .env file.

File: README.md (hunk 2 above)
Comment (zakaprov) [reply README.md:90]: Added this back in babcea9

**Review by azawlocki — APPROVED** [review]
LGTM!

**Pull request merged**